### ISSUE TITLE
When trying to use eb create to deploy this, I get the error Unknown or duplicate parameter: NodeCommand

### ISSUE THREAD

**Issue by Apptation** [yongjhih/docker-parse-server#21]
Uploading docker-parse-server/app-7551-160404_170529.zip to S3. This may take a while.
Upload Complete.
ERROR: Configuration validation exception: Unknown or duplicate parameter: NodeCommand

**Comment by greggmojica** [comment]
Any update on this?

**Comment by yongjhih** [comment]
The eb deployment:
https://s3.amazonaws.com/elasticbeanstalk-samples-us-east-1/eb-parse-server-sample/parse-server-example.zip is provided by https://github.com/ParsePlatform/parse-server-example , I think you should report to https://github.com/ParsePlatform/parse-server-example/issues

**Comment by yongjhih** [comment]
I have no aws account, so i cannot reproduce it and rebuild a deployment package.

**Comment by cleever** [comment]
Same here.

There are an issue in parser serve example:

https://github.com/ParsePlatform/parse-server-example/issues/60

But it was closed with no solution.

**Comment by abhiksingh** [comment]
For those running into the issue with the NodeCommand option setting, you can simply remove the NodeCommand line from the app.config file in the .ebextensions folder.

**Comment by aliasad106** [comment]
@abhiksingh It still does not work for me

**Comment by ryanrca** [comment]
I am reproducing this.  My config does not have a "NodeCommand".
Also, I believe the app.config goes in the .elasticbeanstalk directory, not the .ebextensions folder.

**Comment by jozzhart** [comment]
I had deleted the application from Elastic Beanstalk and neglected to update my config.

In ``config.yml`` in ``.elasticbeanstalk`` check that the application_name exists on Elastic Beanstalk

    global:
      application_name: my-application

My solution was to remove .elasticbeanstalk and simply

    eb init

Otherwise update to an existing application name.

**Comment by aitheenti** [comment]
I just encountered this very same issue. Upon investigation I found that "NodeCommand" is the legacy way to run your application with custom commands.

https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/create_deploy_nodejs.container.html

I removed the ".ebextensions" directory and added a file called "Procfile" to my source directory.

Inside Procfile, try putting the following:

web: npm start
Make sure you update your repository with these changes if necessary before trying to deploy.

Hope this helps!
